src/qt/locale/bitcoin_it.ts

```diff
@@ -548,78 +548,78 @@
     </message>
     <message>
         <source>Wallet is &lt;b&gt;encrypted&lt;/b&gt; and currently &lt;b&gt;unlocked&lt;/b&gt;</source>
-        <translation>Il portamonete è &lt;b&gt;cifrato&lt;/b&gt; ed attualmente &lt;b&gt;sbloccato&lt;/b&gt;</translation>
+        <translation>Il portafoglio è &lt;b&gt;criptato&lt;/b&gt; ed attualmente &lt;b&gt;sbloccato&lt;/b&gt;</translation>
     </message>
     <message>
         <source>Wallet is &lt;b&gt;encrypted&lt;/b&gt; and currently &lt;b&gt;locked&lt;/b&gt;</source>
-        <translation>Il portamonete è &lt;b&gt;cifrato&lt;/b&gt; ed attualmente &lt;b&gt;bloccato&lt;/b&gt;</translation>
+        <translation>Il portafoglio è &lt;b&gt;criptato&lt;/b&gt; ed attualmente &lt;b&gt;bloccato&lt;/b&gt;</translation>
     </message>
     <message>
         <source>&amp;Print paper wallets</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Stampa portafogli di carta</translation>
     </message>
     <message>
         <source>Print paper wallets</source>
-        <translation type="unfinished"></translation>
+        <translation>Stampa portafogli di carta</translation>
```

Comment:
Stampa paper wallets

Reply:
I thought about this for a long, long time. While I agree with you that for younger people or people that know English this sounds much more correct, for the average user that does the extra effort to switch to Italian, "paper wallet" might be a bit too advanced. I imagine my uncle or relatives that would have no clue about what either word mean. "Portafogli di carta" is used also in other wallets for Ethereum/Bitcoin/Litecoin.

To be brief, younger people with modern education in English might not even make the effort to switch to the Italian language; those who do, are those that could have more problems understanding English. Let me know ;)

Reply:
quando un utente si trova di fronte la parola sconosciuta "paper wallet" o non attinente "portafoglio cartaceo" la prima cosa che farà sarà quella di cercare il significato su google. cercando paper wallet troverà subito la risposta più pertinente.

Reply:
Sei piu' ottimista di me ;) Proviamo!